bazel/lib/dependabot/bazel/file_fetcher.rb

```diff
@@ -44,9 +50,94 @@ def fetch_files
 
       sig { override.returns(T.nilable(T::Hash[Symbol, T.untyped])) }
       def ecosystem_versions
-        # TODO: Return supported ecosystem versions
-        # Example: { package_managers: { "bazel" => "1.0.0" } }
-        nil
+        bazel_version = "unknown"
+
+        begin
+          bazelversion_file = fetch_file_if_present(".bazelversion")
+          bazel_version = T.must(bazelversion_file.content).strip if bazelversion_file
+        rescue Dependabot::DependencyFileNotFound
+          # .bazelversion file is optional
+        end
```

Comment:
nit,

`fetch_file_if_present` **sounds** like a retrieval with graceful fallback so we shouldn't need the `rescue`.

If `fetch_file_if_present` does not fallback gracefully, then maybe we should consider changing the behavior or renaming.

In fact, it doesn't use a `rescue` in the `#config_files` method in this class.


```suggestion
        bazelversion_file = fetch_file_if_present(".bazelversion")
        bazel_version = T.must(bazelversion_file.content).strip if bazelversion_file
```